src/components/Logos/RememberLogo/index.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+
+export type EnvType =
+  | 'production'
+  | 'final'
+  | 'alpha'
+  | 'bravo'
+  | 'test'
+  | 'development';
+
+export interface RememberLogoProps {
+  className?: string;
+  color?: 'white' | 'black' | string;
+  height?: number;
+  env?: EnvType;
+}
+
+const getAppEnv = (env?: EnvType): number => {
```

Comment:
```suggestion
const getLogoWidth = (env?: EnvType): number => {
```
맥락상 이정도 네이밍이 맞아 보입니다 ㅎㅎ

Reply:
viewBoxWidth가 맥락에 적절한 것 같아서 요걸로 수정하겠습니다 ㅎㅎ